api/client/amp.go

```diff
@@ -18,14 +23,48 @@ const (
 // Configuration is for all configurable client settings
 type Configuration struct {
 	Verbose bool
+	Github  string
 	Target  string
 	Images  []string
 }
 
+// Save saves the configuration to ~/.ampswarm.yaml
+func (c *Configuration) Save() (err error) {
```

Comment:
This isn't a concern of the `api/client` package which is more about our client-server communication protocol. It should be moved to `cmd/amp/cli`...